apps/ui/src/routes/default.ts

```diff
@@ -67,7 +67,7 @@ export default [
     name: 'my',
     component: My,
     children: [
-      { path: '/home', name: 'my-home', component: Home },
+      { path: '/', name: 'my-home', component: Home },
```

Comment:
Maybe we should keep a redirect from the /home route to the new / route, else people who have a direct link to this page will get a blank page